cpp/deplex/src/deplex/cell_grid.cpp

```diff
@@ -18,6 +18,15 @@
 #include <utility>
 
 namespace deplex {
+
+inline void CellGrid::threadFunction(uint id_thread, config::Config const& config,
```

Comment:
What does "threadFunction" mean? Can we give it a clear name?

Reply:
Fixed